tests/test_views.py

```diff
@@ -177,9 +184,10 @@ def test_user_managers_list_delete_single(self):
             'v1:user-managers-list',
             kwargs={'username': self.users[0].email},
         )
-        self.client.delete(
+        response = self.client.delete(
             '{url}?manager={email}'.format(url=url, email=self.managers[0].email)
         )
+        self.assertEqual(response.status_code, 204)
         query = UserManagerRole.objects.filter(user=self.users[0])
         self.assertEqual(query.count(), 1)
         self.assertNotIn(
```

Comment:
These are good modifications to the test suite. :+1:

Reply:
I had to do them to see the permissions issues in Django 1.8 caused by the DRF test responses!  All the other versions were passing, but not these.